tests/FINDOLOGIC/Tests/Helpers/ParameterBuilderTest.php

```diff
@@ -0,0 +1,855 @@
+<?php
+
+namespace FINDOLOGIC\Tests\Helpers;
+
+use FINDOLOGIC\Definitions\OrderType;
+use FINDOLOGIC\Exceptions\InvalidParamException;
+use FINDOLOGIC\Helpers\ParameterBuilder;
+use PHPUnit\Framework\TestCase;
+
+class ParameterBuilderTest extends TestCase
+{
+    /** @var $parameterBuilder ParameterBuilder */
+    public $parameterBuilder;
+
+    public function setUp()
+    {
+        $this->parameterBuilder = new ParameterBuilder();
+    }
+
+    /**
+     * Returns some shopkeys that might be set by users. Invalid shopkeys are not tested since they should not even get
```

Comment:
Isn't `testExceptionIsThrownIfShopkeyIsInvalid()` testing invalid shopkeys?

Reply:
It seems like this was just a copy & paste issue, also this information is out of date since in every test case after this comment I exactly have a test checking that an exception is thrown if it is invalid.